pages/cart/cart.css

```diff
@@ -0,0 +1,126 @@
+@import url(../../style.css);
+@import url(../product/product.css);
+
+.cart-wrapper {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    height: 77vh;
+    background-color: var(--secondary-grey);
+
+}
+
+.page-header {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 5rem;
+    color: var(--secondary-light);
+    font-weight: 700;
+    font-size: 1.5rem;
+}
+
+.cart-item-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-left: auto;
+    margin-right: 10rem;
+}
+
+/* --------card--------- */
+.horizontal {
+    flex-direction: row;
+    min-width: 450px;
+}
+
+.horizontal .card-header-info {
+    display: none;
+}
+
+.horizontal .card-img {
+    width: 50%;
+}
+
+.horizontal .card-header-img img {
+    border-top-right-radius: unset;
+}
+
+.card-description {
+    width: 50%;
+}
+.product-qty{
+    margin: 0.5rem 0 0.5rem 0;
+    display: flex;
+    align-items: center;
+    gap: 15px;
+}
+.qty-btn{
+    background-color: var(--secondary-grey);
+    border: none;    
+    border-radius: 50%;
+    padding: 0.1rem 0rem 0.1rem 0rem;
+}
+.qty-btn-svg{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.cart-btn {
+    background-color: var(--primary);
+    color: var(--font-primary);
+    margin: 0.5rem 0;
+    width: 88%;
+}
+
+.cart-btn:hover {
+    color: var(--accent);
+}
+
+.cart-price-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+  
+
+}
+
+.cart-price-details {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 70%;
+    padding: 1rem;
+    background-color: rgb(253, 253, 253);
+    border-bottom-left-radius: 5px;
+    border-top-left-radius: 5px;
```

Comment:
suggestion: 
you can use short hand property
/* top-left | top-right | bottom-right | bottom-left */
border-radius: 5px 0 0 5px;